.changeset/kind-wasps-beg.md

```diff
@@ -0,0 +1,6 @@
+---
+'@itwin/itwinui-react': patch
+'@itwin/itwinui-css': patch
+---
+
+Fixed the issue where Dialog moved across the screen when resizing.
```

Comment:
changeset could be removed. there is no change in css, and there is already a react changeset from #1424.

Reply:
Changeset removed